src/client/lib/setup/auth.coffee

```diff
@@ -22,6 +23,7 @@ setupPassport = ->
       headers: 'X-Access-Token': accessToken
       error: (m, err) -> done err
       success: (user) ->
+        user.set 'roles', jwtDecode(accessToken).roles
```

Comment:
(Sorry for later review.)

Is this operation done on every request?

Reply:
No worries. Let me check that.

Reply:
Looks like it's done once per session:
```
positron_1                | GET /login 302 1.106 ms - 0
positron_1                | GET /api/users/me 200 132.749 ms - 164
positron_1                | setting user.roles
positron_1                | GET /api/channels/5fd6d1f9fa338d0e1d8ac143 200 4.184 ms - 101
```